Fix links to node troubleshooting & monitoring guides

mablr opened the pull request:
**Description**
Fix links to node troubleshooting & monitoring guides

**Tests**
N/A

**Additional context**
Follow-up for #1802 and #1811

**Metadata**
N/A

mablr committed:
fix links for Node Troubleshooting & Monitoring Guides

sbvegan commented on node-operators/reference/architecture/architecture.mdx:
```diff
@@ -45,5 +45,5 @@ Legacy Geth is **not** required and is typically only necessary if you want to m
-*   If you've already got your node up and running, check out the [Node Metrics and Monitoring Guide](./management/metrics) to learn how to keep tabs on your node and make sure it keeps running smoothly.
-*   If you run into any problems, please visit the [Node Troubleshooting Guide](./management/troubleshooting) for help.
+*   If you've already got your node up and running, check out the [Node Metrics and Monitoring Guide](/operators/node-operators/management/metrics) to learn how to keep tabs on your node and make sure it keeps running smoothly.
+*   If you run into any problems, please visit the [Node Troubleshooting Guide](/operators/node-operators/management/troubleshooting) for help.
```
these links are unfortunately also wrong

mablr replied:
Oh ok, thanks for the quick review!

I actually hesitated on these, and took as reference this page where the link is ok:
https://github.com/ethereum-optimism/docs/blob/103dd6c39a910b577a2c9a2dfff1e8e9be8f2fce/node-operators/reference/architecture/rollup-node.mdx?plain=1#L135-L136
https://docs.optimism.io/node-operators/reference/architecture/rollup-node#next-steps

I assume the correct ones are respectively:
- "Node Metrics and Monitoring Guide": /node-operators/guides/monitoring/metrics
- "Node Troubleshooting Guide": /node-operators/guides/troubleshooting

sbvegan replied:
Yeah I think that's right. Everything got shuffled around recently because we moved to a new framework. I'm probably going to do some more sweeping changes soon. If you follow the filesystem paths and what's in the docs.json you should be good

mablr replied:
Ok, fix applied 👍

mablr committed:
fix links in architecture.mdx

/operators/node-operators/management/metrics -> /node-operators/guides/monitoring/metrics

/operators/node-operators/management/troubleshooting -> /node-operators/guides/troubleshooting
